frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -77,6 +77,7 @@ export const queueQuery = gql`
     organization {
       testingFacility {
         id
+        name
```

Comment:
**question:** You've added these attributes to the Queue query, but it looks like you're getting these values from the redux store. Is this change needed?

Also just as a heads up, there's a long-term plan to do away w/ the `org { facilities }` query that's made here adjacent to the Queue query – so this potentially adds a tiny little cost to that future tech debt

Reply:
The _organization_ name comes from the Redux store but as of the most recent commit the facility name doesn't. That said, it should! See f89b5e1.

Reply:
Thanks for updating this! Could we use `useSelectedFacility` instead of referencing the store by hand? Just DRYer, I think

Reply:
Neat! 👀 I'm not sure I've used that before. Updated in 15822c7.